<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>runPIHM</class>
 <widget class="QDialog" name="runPIHM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PIHM</string>
  </property>
  <widget class="Q3TextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>215</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>26</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Input Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>76</y>
      <width>102</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>24</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFile">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>76</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>76</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>This creates a new Folder (Simul__) for each simulation run where all the model input and output is stored.</string>
    </property>
    <property name="text">
     <string>Create New Folder</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>21</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>146</y>
     <width>531</width>
     <height>56</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>18</y>
      <width>216</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>13</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>13</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string> Stop </string>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>13</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string> RUN </string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
